### PR TITLE
Test refactoring

### PR DESCRIPTION
**Pull request by johnotander**
While attempting to reproduce the bug from https://github.com/jxnblk/basscss/issues/68 I did some refactoring to the test suite. I was unsuccessful in reproducing said bug, however, I thought you might like to incorporate this test suite.

Thanks for another awesome project!

### CODE REVIEW

**Comment by jxnblk** [comment]
Rad. Thanks for putting this together. I've been meaning to learn how to write actual tests, so this is super helpful.

File: test/fixtures/variables.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:root {
+  --some-var: #fff;
+  --some-other-var: var(--some-other-var);
```
Comment (jxnblk) [inline test/fixtures/variables.css]: Should the value here be `var(--some-var)`? Just want to be sure I understand what this is testing.

Comment (johnotander) [inline test/fixtures/variables.css]: Ah, yeah. Whoops. :blush:

**Comment by johnotander** [comment]
I'm glad it can be of use. I've fixed the variables test to use the correct variable assignment. Nice catch!

**Pull request merged**